ui/etc/release.conf

```diff
@@ -8,10 +8,10 @@
 # it is provided by or on behalf of EMC.
 
 release_name="ECS Community Edition"
-release_version="3.1.0.0"
+release_version="3.1.0.1"
 release_product="ECS Software"
 release_artifact="emccorp/ecs-software-3.1.0"
-release_tag="latest"
+release_tag="hf1"
```

Comment:
Is this just for testing until we promote hf1 to latest?

Reply:
Email said 3.1.0.1 replaces 3.1.0.0 immediately.  I took that to mean immediately.  So, my plan is to promote 3.1.0.1 to latest ASAP.

Reply:
this tag is for hf1 specifically.  I'm using version specific tags now.  HF1 will also be tagged "latest" once it's live.

Reply:
In fact, it's already tagged latest, so 2.5.0 instances are getting 3.1.0.1 now as well.

Reply:
coolio